### PR TITLE
Moving syntax vizualizer to depend on 1.0 binaries for compatibility …

### PR DESCRIPTION
**Pull request by jmarolf**
Moving syntax vizualizer to depend on 1.0 binaries for compatibility with VS RTM

### CODE REVIEW

**Comment by jmarolf** [comment]
@dotnet/roslyn-ide @dotnet/roslyn-infrastructure

File: src/Tools/Source/SyntaxVisualizer/SyntaxVisualizerControl/project.json [hunk 1]
```diff
@@ -1,8 +1,11 @@
...
+﻿{
+  "dependencies": {
+    "Microsoft.CodeAnalysis.CSharp": "1.0.0",
+    "Microsoft.CodeAnalysis.VisualBasic": "1.0.0",
+    "Microsoft.CodeAnalysis.Analyzers": "1.1.0"
+  },
+  "frameworks": {
+    "net45": { }
+  },
+  "runtimes": { "win7": { } }
```
Comment (Pilchie) [inline src/Tools/Source/SyntaxVisualizer/SyntaxVisualizerControl/project.json]: This won't work on pre update1 (at least if the project is set to copy nuget implementations).  To support that, you should add `win-anycpu`

File: Restore.cmd [hunk 2]
```diff
@@ -15,6 +15,10 @@ call %NugetExe% restore -verbosity quiet "%~dp0build\ToolsetPackages\project.jso
 echo Restoring packages: Samples
 call %NugetExe% restore -verbosity quiet "%~dp0src\Samples\Samples.sln" -configfile "%~dp0nuget.config" || goto :RestoreFailed
 
+echo Restoring packages: Syntax Vizualizer Components
+call %NugetExe% restore -verbosity quiet "%~dp0src\Tools\Source\SyntaxVisualizer\SyntaxVisualizerControl\project.json" -configfile "%~dp0nuget.config" || goto :RestoreFailed
+call %NugetExe% restore -verbosity quiet "%~dp0src\Tools\Source\SyntaxVisualizer\SyntaxVisualizerDgmlHelper\project.json" -configfile "%~dp0nuget.config" || goto :RestoreFailed
```
Comment (Pilchie) [inline Restore.cmd]: Is this the right place to do this?  If these are already part of the internal setup.proj, don't we restore that?

**Comment by Pilchie** [comment]
A couple of questions, but generally :+1:

**Pull request merged**